packages/web/src/components/remove/remove-liquidity-select-list-item/RemoveLiquiditySelectListItem.tsx

```diff
@@ -27,27 +27,37 @@ interface TooltipProps {
 }
 
 const TooltipContent: React.FC<TooltipProps> = ({ position, disabled }) => {
+  const renderTokenValue = (imgUri: string, tokeSymbol: string, token: TokenModel, tokenBalance: bigint ) => {
+    const tokenBalanceByTokenDecimal = BigNumber(makeDisplayTokenAmount(token, tokenBalance) || 0).toFormat();
+
+    return <div>
```

Comment:
Create a styled component instead of a `div`.
The default styling of `div` makes it difficult to predict what styles will be applied. 

I have the same problem with the `title` side, which is not a change, 
could you fix that if you don't mind?

Reply:
okay let me check